.github/workflows/release.yml

```
@@ -84,4 +80,32 @@ jobs:
           packages-dir: dist
           print-hash: true
           verbose: true
-          # repository-url: https://test.pypi.org/legacy/
+          repository-url: https://test.pypi.org/legacy/
+
+  public-pypi:
+    runs-on: ubuntu-latest
+    needs: public-testpypi
+    if: ${{ needs.release.outputs.released == 'true' }}
```

actionlint (1.7.8)

property "release" is not defined in object type {public-testpypi: {outputs: {}; result: string}}

(expression)